apps/base/cms-rucio-common.yaml

```diff
@@ -1,15 +1,16 @@
 image:
-  tag: release-1.19.5
+  tag: release-1.19.5 #NOTE: Is this rucio image version?
 
 config:
     accounts:
         special_accounts: "tier0"
     policy:
-        permission: "cms"
-        schema: "cms"
+        package: "CMSRucioPolicy"
+        # permission: "cms"
```

Comment:
Please get rid of commented out lines. This leave code and configs very hard to follow after a time.